rlapy/tests/test_comps/test_qb.py

```diff
@@ -110,20 +271,266 @@ def test_exact(self):
             blk=4,
             overwrite_a=False
         )
-        # Code from here onward is copied from TestQB1.
-        #   That's undesirable.
+        alg_tol = np.NaN
```

Comment:
This is a good opportunity for an algorithm-specific test. Verify that the matrix ``ath.A`` is the same before and after a call to ``self.run_batch_exact``. (Make a copy before the function call and compare to after the function call.) It suffices to test this on a single matrix.